test/xhtml/core/utils/is-xhtml.js

```diff
@@ -0,0 +1,7 @@
+describe('axe.utils.isXHTML', function () {
```

Comment:
I don't think you need this file, it's redundant with `test/core/utils/is-xhtml.js`.

Reply:
see explanation in the general comment below 👇

Reply:
specifically, this will test that the `document` **from the current fixture** is XHTML. I copied the pattern from aXe's pre-existing [`is-html5.js` tests](https://github.com/dequelabs/axe-core/blob/develop/test/commons/dom/is-html5.js#L14)